docs/cloud/cloud-changelog.md

```diff
@@ -85,3 +85,36 @@ AlmaLinux Generic Cloud images have now unified - BIOS and UEFI - boot support.
 **Impacted architecture: x86_64**
 
 AlmaLinux OpenNebula images have now unified - BIOS and UEFI - boot support. The download URLs of the UEFI images are symlinked to the current image for compatibility.
+
+## Oracle Cloud Infrastructure (OCI)
+
+### AlmaLinux OS versions **9.5-20250205**
+
+* Fixes:
+  * The OCI-specific kernel command-line parameters are ineffective, resulting in a broken boot process on bare metal instances that use iSCSI disks.
+
+* New packages were added:
+  * `langpacks-en`: Adds `en_US.UTF-8` setting it as a default locale.
+
+### AlmaLinux OS versions **9.5-20250205** and AlmaLinux OS versions **8.10-20250215**
```

Comment:
@LKHN since version **9.5-20250205** is also already mentioned above, how about moving the fix/new package here but highlighting that it's only for 9.5?